src/models/searchservice/response/HighlightedFields.ts

```diff
@@ -0,0 +1,47 @@
+import { HighlightedValue } from './HighlightedValue';
+
+/**
+ * A mapping of the values emphasized by the Answers API.
+ *
+ * @remarks
+ * The structure of HighlightedFields matches the structure of the corresponding rawData of a {@link Result}.
+ *
+ * @example
+ * If a user searches for 'pet' and a description field in the results contains the value 'A pet store',
+ * the API will likely match the word 'pet'.
```

Comment:
In the generated docs, the examples are a little confusing. Looking at the docs, I saw the JS snippet from the second example and immediately assumed it was related to the first example, which had no snippet. Then, I saw the two examples were unrelated. The first example doesn't have a sample JS snippet and the second doesn't have much introductory explanation.

Reply:
ah yeah I can unify the two